src/server/handlers/robots-handler.ts

```diff
@@ -6,7 +6,9 @@ export default async ({ res }: { res: Response }) => {
   res.send(`User-Agent: *
 Disallow: /login
 Disallow: /login_reset
+DIsallow: /signup
 Disallow: /settings
+Disallow: /instances
```

Comment:
Allowing /instances makes some sense for bots to discover other Fediverse sites.